onnxruntime/python/tools/tensorrt/perf/ort_build_latest.py

```diff
@@ -0,0 +1,35 @@
+import os
```

Comment:
let's avoid usage of hardcoded values in scripts.
e.g. /home/hcuser etc. 
we want to keep this scripts generic so they can be executed in any environment. 
or if there are any assumptions about environment it needs to be documented.

Reply:
The purpose of ort_build_latest.py is to use as much less time as possible to build/install the latest ORT python wheel file.
Therefore, indicating a fixed repo that is "almost" up-to-date and being built frequently will largely reduce the build time. 
I will make this value, e.g. /home/hcsuser/, configurable or a command argument to avoid hardcoded. Also I will note this in README.